index.js

```diff
@@ -128,11 +134,53 @@ async function task(options = { quiet: true }) {
   if (options.quiet) { // for running with setInterval
     process.stdout.write('.')
   } else { // for individual runs, or first of a setInterval run
-    console.log(`No available appointments for months ${MONTHS.join(', ')} in ${EMBASSY.displayName} (${EMBASSY.postCode})`)
+    console.log(`No available appointments for months ${DATES.flatMap(e => e.MONTH).join(', ')} in ${EMBASSY.displayName} (${EMBASSY.postCode})`)
+  }
+}
+
+function initialize() {
+
+  let currentDate = new Date();
```

Comment:
lines 143-161 should ideally be abstracted into a separate function, so that it's clear that the intent of the block is to create/mutate an array of dates.